kybra/compiler/kybra_generate/src/canister_method/post_upgrade_method/rust.rs

```diff
@@ -28,19 +28,21 @@ pub fn generate(
     );
 
     Ok(quote! {
-        #randomness
+        ic_cdk_timers::set_timer(std::time::Duration::from_secs(0), move || {
```

Comment:
Comment about this here, link to or explain dfx chunk uploading and named/default callbacks as potential solutions in the future